SECP-service/src/main/java/com/visucius/secp/models/User.java

```diff
@@ -85,16 +93,18 @@
     public  User () {
 
     }
-    public User(String firstname, String lastname, String userName, String email, String password) {
+    public User(String firstname, String lastname, String userName, String displayname, String email, String password, String avatar_url) {
```

Comment:
don't change constructor. use the setters method

Reply:
Ya we need to be really careful when changing constructors it can break stuff.